spec/models/initiative_spec.rb

```diff
@@ -17,13 +17,20 @@
       end
     end
   end
-end
 
-describe InitiativesController, type: :controller do
-  let!(:initiative) { create(:initiative) }
+  describe InitiativesController, type: :model do
+    let!(:initiative) { create(:initiative) }
+    it "assign incompleted initiatives" do
+      incompleted_initiatives = Initiative.incompleted
+      expect(incompleted_initiatives).to eq([initiative])
+    end
+  end
 
-  it "deletes the initiative" do
-    expect{ delete :destroy, params: {id: initiative.id}}.to change(Initiative,:count).by(-1)
-    expect(response).to redirect_to root_path
+  describe InitiativesController, type: :model do
```

Comment:
Should this model spec describe an `InitiativesController`? We're already describing the model on line 5 and then again on line 6. Maybe manipulating the completion is best as a `context`.